README.md

```diff
@@ -67,41 +62,43 @@ If you experience slow launch times of fastlane, try running:
 
     gem cleanup
 
-System Requirements: `fastlane` requires Mac OS X or Linux with Ruby 2.0.0 or above.
+**System Requirements:** `fastlane` requires Mac OS X or Linux with Ruby 2.0.0 or above
 
-If you want to take a look at a project already using `fastlane`, check out the [fastlane-examples](https://github.com/fastlane/examples) with `fastlane` setups by Wikipedia, Product Hunt, MindNode and more.
+If you'd like to take a look at a project already using `fastlane` check out [fastlane-examples](https://github.com/fastlane/examples) which includes `fastlane` setups by Wikipedia, Product Hunt, MindNode, and more.
 
 ## Quick Start
 
-The setup assistant will create all the necessary files for you, using the existing app metadata from iTunes Connect or Google Play.
+`fastlane`'s setup assistant creates all the necessary files to get you started, using existing app metadata from iTunes Connect or Google Play.
 
 - `cd [your_project_folder]`
 - `fastlane init`
-- Follow the setup assistant, which will set up `fastlane` for you
-- Further customise the `Fastfile` with [actions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md).
+- Follow the assistant's prompts
+- Further customise the Ruby based `Fastfile` with additional [actions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md)
 
 For more details, check out the [fastlane guide](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Guide.md).
 
-## Available commands
+## Available Commands
 
-Usually you'll use fastlane by triggering individual lanes:
+Typically you'll use `fastlane` by triggering your individual lanes:
 
     fastlane [lane_name]
 
-#### Other commands
+#### Other Commands
 
 - `fastlane actions`: List all available `fastlane` actions
 - `fastlane action [action_name]`: Shows a more detailed description of an action
 - `fastlane lanes`: Lists all available lanes with description
 - `fastlane list`: Lists all available lanes without description
-- `fastlane new_action`: Create a new action (integration) for fastlane
+- `fastlane new_action`: Create a new action *(integration)* for fastlane
 
 ## [`fastlane`](https://fastlane.tools) Toolchain
 
-Additionally to the `fastlane` commands, you now have access to these `fastlane` tools:
+In addition to `fastlane`'s commands, you also have access to these `fastlane` tools:
 
 - [`deliver`](https://github.com/fastlane/fastlane/tree/master/deliver): Upload screenshots, metadata, and your app to the App Store
+- [`supply`](https://github.com/fastlane/fastlane/tree/master/supply): Upload your Android app and its metadata to the Google Play Store
```

Comment:
"Upload screenshots, metadata and your app to Google Play. Select tracks for builds and promote builds to production"